packages/explorer/src/components/AccountSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import { Address, formatEther } from "viem";
+import { useBalance } from "wagmi";
+import { ACCOUNTS, CONFIG } from "../consts";
+import { useStore } from "../store";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "./ui/Select";
+import { TruncatedHex } from "./ui/TruncatedHex";
+
+function AccountSelectItem({
+  address,
+  name,
+}: {
+  address: Address;
+  name: string;
+}) {
+  const balance = useBalance({
+    address,
+    query: {
+      refetchInterval: CONFIG.BALANCES_REFETCH_INTERVAL,
```

Comment:
does this need to be configurable?

Reply:
Generally, I like to use config for such values in case it repeats across several similar situations. Then I can configure all of them from one place, and stay consistent. But after having a second though, it's not necessary here, removed.